docs/event-types.mdx

```diff
@@ -110,17 +110,17 @@ curl -X POST "https://api.svix.com/api/v1/event-type/" \
 
 ## Bulk creating event types
 
-The easiest way to bulk-create event types is by just writing a simple script to load a `CSV` or a `JSON` file with the event types and make the API requests.
+The easiest way to bulk-create event types is by just writing a simple script to load a [`TSV` file](https://en.wikipedia.org/wiki/Tab-separated_values) or a `JSON` file with the event types and make the API requests.
 
-Here is an example with a `CSV` file (without headers). Here is an example `CSV`:
+Here is an example `TSV` file (without headers) of events:
 
-```
-user.created,A user has been created
-user.removed,A user has been removed
-user.changed,A user has changed
+```tsv
+user.created	A user has been created
+user.removed	A user has been removed
+user.changed	A user has changed
```

Comment:
docusaurus or prisim is converting these tabs to spaces, I haven't been able to figure out how to get it to not do this yet, but other than that it should be ready

Reply:
Let's use a pipe-delimited CSV then.

Reply:
Fixed.